projects/plugins/jetpack/extensions/blocks/ai-chat/loading.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Circle } from '@wordpress/components';
+export default function Loading() {
```

Comment:
Should we use the core spinner instead of bringing our own loading indicator?
https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/spinner#readme

Reply:
I will leave this to Agora to make a call on. My thinking at the time was it needed something different to the standard loading spinner in order to give the impression that rather than there being a delay with loading, the AI just needed time to think 😄, which is sort of the case. 

This may not be such an issue if we do switch to streaming from the API as the response will start arriving much quicker in chunks.